server_worker/server_worker.py

```diff
@@ -195,7 +195,7 @@ def import_data_by_ibis(
         if columns_types:
             columns_types_pd = convert_type_ibis2pandas(columns_types)
         t0 = time.time()
-        if files_limit > 1:
+        if files_limit > 1 or isinstance(data_files_names, list):
```

Comment:
```suggestion
        if files_limit > 1 or isinstance(data_files_names, (list, tuple)):
```
no need to limit to just `list`, `tuple` should be accepted as well. actually, any sequence if we want to go deep enough, but checking just those two should be enough